Dockerfile

```diff
@@ -19,7 +19,8 @@ RUN npm -w @filecoin-plus/application install
 RUN npm -w @filecoin-plus/application run build 
 
 # Expose the port the app runs on
-EXPOSE 3000
```

Comment:
does this port need to change?

Reply:
Not necessarily. In the dockerized development environment the front end runs on port 3000 and it looks for the api on port 3001, so in that sense yes, _something_ needs to change.

But if it's better for other environments (especially Prod!) we could put this back so that the API runs on 3000 in its local env and then just have docker-compose map it to port 3001 to dev builds